library/core/tests/slice.rs

```diff
@@ -249,6 +249,40 @@ fn test_chunks_nth() {
     assert_eq!(c2.next(), None);
 }
 
+#[test]
+fn test_chunks_next() {
```

Comment:
Actually, I actually didn't need to add this one, since I didn't change Chunks::next code, but it doesn't hurt (the existing tests are slightly less explicit).

The others are all the functions I changes though, I think.